src/includes/class-sesamy-api-endpoint.php

```diff
@@ -92,8 +92,6 @@ public function sesamy_post_ep( $request ) {
 
 		$post = get_post( $request['id'] );
 
-		print_r($post); exit;
```

Comment:
This early response prevented the flow from working properly (instead of returning the content if the article is public or verifying the JWT if the article is locked).
I see that this change was implemented by you @PradeepWeb77 , any idea why? Maybe because there was an error with `Basic` token as fixed in line 105: https://github.com/sesamyab/wordpress-sesamy/pull/106/files#r1572096936 ?

Reply:
This was also making the content to look like this: https://files.slack.com/files-pri/TSPDCEFU6-F06UXGEE0SE/captura_de_pantalla_2024-04-18_124413.png once unlocked